code/modules/client/preferences.dm

```diff
@@ -588,6 +590,68 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 				dat += "</td>"
 			dat += "</tr></table>"
 
+		if (3) // Keybindings
+			dat += "<center><a href='?_src_=prefs;preference=hotkeys'>[(hotkeys) ? "Hotkeys" : "Default"]</a>"
+			dat += "<a href='?_src_=prefs;preference=reset_bindings'>Reset to default</a></center>"
+			if(hotkeys)
+				var/button
+				var/button_bound
+
+				dat += "<table><tr><td width='340px' height='300px' valign='top'>"
+				dat += "<h2>Client</h2>"
+				BUTTON_KEY_MOVEMENT("Move North (up)", ACTION_MOVENORTH, NORTH)
+				BUTTON_KEY_MOVEMENT("Move West (left)", ACTION_MOVEWEST, WEST)
+				BUTTON_KEY_MOVEMENT("Move South (down)", ACTION_MOVESOUTH, SOUTH)
+				BUTTON_KEY_MOVEMENT("Move East (right)", ACTION_MOVEEAST, EAST)

```

Comment:
The actions available per type should be determined by the code not changed manually here.